APITesters/SwiftAPITester/SwiftAPITester/PurchasesAPI.swift

```diff
@@ -0,0 +1,198 @@
+//
+//  Copyright RevenueCat Inc. All Rights Reserved.
+//
+//  Licensed under the MIT License (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//      https://opensource.org/licenses/MIT
+//
+//  PurchasesAPI.swift
+//
+//  Created by Madeline Beyl on 8/25/21.
+
+import Foundation
+import RevenueCat
+import StoreKit
+
+func checkPurchasesAPI() {
+    // initializers
+    let purch = Purchases.configure(withAPIKey: "")
+    Purchases.configure(withAPIKey: "", appUserID: nil)
+    Purchases.configure(withAPIKey: "", appUserID: "")
+
+    Purchases.configure(withAPIKey: "", appUserID: "", observerMode: false)
+    Purchases.configure(withAPIKey: "", appUserID: nil, observerMode: true)
+
+    Purchases.configure(withAPIKey: "", appUserID: "", observerMode: true, userDefaults: nil)
+    Purchases.configure(withAPIKey: "", appUserID: nil, observerMode: true, userDefaults: nil)
+    Purchases.configure(withAPIKey: "", appUserID: "", observerMode: true, userDefaults: UserDefaults())
+    Purchases.configure(withAPIKey: "", appUserID: nil, observerMode: true, userDefaults: UserDefaults())
+
+    let finishTransactions: Bool = purch.finishTransactions
+    let delegate: PurchasesDelegate? = purch.delegate
+    let appUserID: String = purch.appUserID
+    let isAnonymous: Bool = purch.isAnonymous
+
+    print(finishTransactions, delegate!, appUserID, isAnonymous)
+
+    checkStaticMethods()
+    checkIdentity(purchases: purch)
+    checkPurchasesSubscriberAttributesAPI(purchases: purch)
+    checkPurchasesPurchasingAPI(purchases: purch)
+}
+
+var type: PeriodType!
```

Comment:
`type` has a special meaning in Swift, perhaps periodType is better in this context

Reply:
💯